indico/modules/rb_new/client/js/common/bookings/BookingObjectLink.jsx

```diff
@@ -15,20 +15,31 @@
  * along with Indico; if not, see <http://www.gnu.org/licenses/>.
  */
 
+import _ from 'lodash';
 import React from 'react';
 import PropTypes from 'prop-types';
 import {Message, Placeholder, Segment, Icon} from 'semantic-ui-react';
-import getBookingEventDataURL from 'indico-url:rooms_new.booking_event_data';
+import getLinkedObjectDataURL from 'indico-url:rooms_new.linked_object_data';
```

Comment:
move that import to the top (separate block of imports) like we do e.g. in the actions

Reply:
Then we should also change this on `SpriteImage.jsx`, `RoomSelector.jsx` and `RoomStats.jsx`